pype/plugins/maya/publish/extract_playblast.py

```diff
@@ -110,6 +110,9 @@ def process(self, instance):
         if not instance.data.get("keepImages"):
             tags.append("delete")
 
+        # Add camera node name to representation data
+        camera_node_name = pm.ls(camera)[0].nodeName()
```

Comment:
Not sure if this is right way how to get the name.
Also not sure if this is right plugin to do this, was thinking that this may be done in `Collect Review Data` where camera is collected to instance data.

Reply:
you actually don't have to do this at all. camera name is already collected in the instance. you just need to grab it and use it.

Reply:
> you just need to grab it and use it.

Do it, please :)